pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -175,13 +175,16 @@ func getTaskRunByName(t *testing.T, taskRuns map[string]*v1beta1.TaskRun, expect
 	return tr
 }
 
-// getTaskRunsForPipelineRun returns the set of TaskRuns associated with the input PipelineRun.
-// It will fatal the test if an error occurred.
-func getTaskRunsForPipelineRun(ctx context.Context, t *testing.T, clients test.Clients, namespace string, prName string) map[string]*v1beta1.TaskRun {
+// getTaskRunsForPipelineRun returns the set of TaskRuns associated with the input PipelineRun and optional
+// input of a PipelineTask Label selector. It will fatal the test if an error occurred.
```

Comment:
would it make sense to create another helper function `getTaskRunsForPipelineTask`?

Reply:
Oh okay, yes I just created this helper func instead.

Reply:
Thanks! Can you fix the docstrings please?

Optional: getTaskRunsForPipelineTask can call getTaskRunsForPipelineRun and filter its results